packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -517,13 +517,13 @@ export class MirrorNodeClient {
         return `${constants.CACHE_KEY.GET_CONTRACT}.valid.${contractIdOrAddress}`;
     }
 
-    public getIsValidContractCache(contractIdOrAddress): any {
+    public async getIsValidContractCache(contractIdOrAddress): Promise<any> {
```

Comment:
`await` not necessary with `get`
```suggestion
    public getIsValidContractCache(contractIdOrAddress): any {
```

Reply:
Yes, removed the async and promise